packages/synapse-react-client/package.json

```diff
@@ -249,8 +249,6 @@
     "build:umd": "vite build -c vite.config.umd.ts",
     "storybook:build": "pnpm storybook build",
     "type-check": "tsc --build",
-    "link-src": "pnpm link && cd ./node_modules/react && pnpm link && cd ../react-router && pnpm link && cd ../react-router-dom && pnpm link && cd ../../",
-    "unlink-src": "pnpm unlink && cd ./node_modules/react && pnpm unlink && cd ../react-router && pnpm unlink && cd ../react-router-dom && pnpm unlink && cd ../../",
```

Comment:
These haven't done anything since we moved this and portals to a pnpm monorepo.

Linking might still be useful for SWC, but today that would use `yarn`, not `pnpm`